cmd/wallet-web/src/pages/Store.vue

```diff
@@ -66,15 +66,15 @@
                   border-b border-neutrals-dark
                 "
               >
-                <label for="select-key" class="mb-1 text-sm font-bold text-neutrals-dark"
-                  >Select Vault</label
-                >
-                <select
-                  v-model="selectedVault"
-                  class="mb-1 w-full max-w-full text-base text-neutrals-dark bg-neutrals-lilacSoft"
-                >
-                  <option>Default Vault</option>
-                </select>
+                <label for="select-key" class="mb-1 text-sm font-bold text-neutrals-dark">{{
+                  $t('Vaults.selectVault')
+                }}</label>
+                <custom-select
+                  id="valutOptions"
+                  :options="vaults"
+                  :default="$t('Vaults.defaultVault')"
+                  @selected="vaultSelected"
+                ></custom-select>
```

Comment:
`$t` is deprecated in the `vue-i18n-next`, you have to use a hook to expose `t` function in the `setup` lifecycle function and use it in template. For example, see https://github.com/trustbloc/wallet/blob/main/cmd/wallet-web/src/components/CredentialDetails/Banner.vue#L122 and https://github.com/trustbloc/wallet/blob/main/cmd/wallet-web/src/components/CredentialDetails/Banner.vue#L144-L147

Reply:
updated